test_vm/vendor/cookbooks/dbfit_test/recipes/sqlserver.rb

```diff
@@ -0,0 +1,44 @@
+remote_file '/etc/yum.repos.d/mssql-server.repo' do
+  source 'https://packages.microsoft.com/config/rhel/7/mssql-server.repo'
+  action :create
+end
+
+package 'mssql-server'
+
+ENV['SA_PASSWORD'] = 'DbFit000!'
+ENV['ACCEPT_EULA'] = 'Y'
+
+service 'stop mssql-server' do
+  service_name 'mssql-server'
+  action :stop
+end
+
+execute 'configure SQL Server' do
+  command '/opt/mssql/bin/mssql-conf setup'
+  action :run
+  not_if 'systemctl status mssql-server | grep active (running)'
+end
+
+service 'start mssql-server' do
+  service_name 'mssql-server'
+  action :start
+end
+
+remote_file '/etc/yum.repos.d/prod.repo' do
+  source 'https://packages.microsoft.com/config/rhel/7/prod.repo'
+  action :create
+end
+
+package 'mssql-tools'
+
+sql_dir = File.join(node['dbfit']['project_root'], "test_vm/scripts/sqlserver/sql")
+
+execute 'create databases' do
+  command "/opt/mssql-tools/bin/sqlcmd -S localhost -U sa -P DbFit000! -b -i #{sql_dir}/create-databases-sqlserver.sql"
```

Comment:
Is it OK that we don't have some kind of `not_if` expression here? What would happen if we re-run VM provisioning?